thonny/tktextext.py

```diff
@@ -34,6 +34,15 @@ def __init__(self, master=None, cnf={}, read_only=False, suppress_events=False,
         self._original_mark = self._register_tk_proxy_function("mark", self.intercept_mark)
         self.bind("<Control-Tab>", self._redirect_ctrl_tab, False)
 
+        # Reference: https://wiki.tcl-lang.org/page/Modern+Bindings+for+the+Text+Widget
```

Comment:
Is all of this required for each instance of Text?

Reply:
If I remember correctly the `self.bindtags` is required every time a new text instance is created.